modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -49,6 +49,90 @@ A new transaction is started every time the scope is executed.
+* Always Begin (`ALWAYS_BEGIN`): this time a new transaction is began for the execution of the try scope. After the error is raised (and not handled), the new transaction is rollbacked, which means that the `jms:publish` operation will not commit and thus the message is not actually sent to the `test.out` queue. Then, the transaction of the flow (which was began by the message source) will also be rollbacked, and message will be available again in the `test.in` queue. 
+* Begin or Join (`BEGIN_OR_JOIN`): in this case, when the error is raised, it is not inmediately rollbacked. Since the flow could handle the raised error (therfore, commit the transaction) the message sent by the `jms:publish` could still be sent. This time, since the error is not handled with an `on-error-continue`, then the entire transaction is rollbacked and the result is the same as in the case of Always Begin. 
+
+Suppose now we add an Error Handler at flow level:
+
+[source,xml,linenums]
+----
+<flow name="someFlow">
+	<jms:listener config-ref="JMS_Config" destination="test.in" transactionalAction="ALWAYS_BEGIN"/>
+	<!-- Processors -->
+	<try transactionalAction="?">
+		<!-- Processors -->
+		<!-- Join if possible is the default value for jms:publish operation --> 
+		<jms:publish config-ref="JMS_Config" destination="test.out"/>
+		<raise-error type="APP:SOME"/>
+		<!-- More Processors -->
+	</try>
+	<!-- Processors -->
+	<error-handler>
+		<on-error-continue/>
+	</error-handler>
+</flow>
+----
+
+We analyze this new scenario: 
+* Ignore (`INDIFFERENT`): Once again, the transaction is _suspended_, then message is sent in the `jms:publish` operation regardless of the error caused by the `raise-error` processor. Then, as the error is handled in the `on-error-continue` Error Handler, the transaction is commited and the message is consumed. 
+* Always Begin (`ALWAYS_BEGIN`): Since the transaction that runs within the Try Scope is a different one from the flow, this one is rollbacked (since it is not handled) and the error propagated and handled at flow level. Therefore, the message will still be consumed, but the message sent with the `jms:publish` operation not (that transaction is rollbacked)
+* Begin or Join (`BEGIN_OR_JOIN`): As mentioned in the previous example, the commit of the `jms:publish` message depends on the flow. Since the error is now handled by an `on-error-continue` Error Handler, the transaction will be commited. Thus, both the message read from the `jms:listener` source will consumed and the message sent with `jms:publish` will actually be sent.
```

Comment:
will be commited > is committed
will consumed > is consumed
will actually be sent > is actually sent